beetsplug/fetchart.py

```diff
@@ -505,12 +505,18 @@ def get(self, album, plugin, paths):
                             payload['term'])
             return
 
+        if self._config['high_resolution'].get():
```

Comment:
```suggestion
        if self._config['high_resolution']:
```

The Confuse library is smart enough to turn this into a `bool` automatically without calling `get()`.

Reply:
Ah, good to know. Thanks for reviewing so quickly!